<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pyqtuino</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/usb.png</normaloff>:/icons/usb.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Text">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>30</red>
             <green>159</green>
             <blue>131</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="BrightText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="ButtonText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>30</red>
             <green>159</green>
             <blue>131</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="HighlightedText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <attribute name="title">
        <string>Com</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>231</width>
          <height>361</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>com setting</string>
        </property>
        <widget class="QComboBox" name="comboBoxPort">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>com</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>baurd</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxBaud">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>first</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxCheckSum">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>130</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>data</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxBits">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>180</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>230</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>stop</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxStopBits">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>230</y>
           <width>121</width>
           <height>31</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButtonOpenSerial">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>201</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>open</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonSendData">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>210</y>
          <width>61</width>
          <height>171</height>
         </rect>
        </property>
        <property name="text">
         <string>send</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditReceived">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>541</width>
          <height>181</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEditSent">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>210</y>
          <width>471</width>
          <height>171</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'PMingLiU'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="main_splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="sim_start">
              <property name="text">
               <string>start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sim_stop">
              <property name="text">
               <string>stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="simtree">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::InternalMove</enum>
              </property>
              <property name="defaultDropAction">
               <enum>Qt::MoveAction</enum>
              </property>
              <column>
               <property name="text">
                <string>1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QHBoxLayout" name="painter_layout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>custom</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>631</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="graphyViewModule">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="treeWidget" native="true">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTreeWidget" name="tree">
              <attribute name="headerShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>compoment</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuchange">
     <property name="title">
      <string>change</string>
     </property>
     <addaction name="actionbasic"/>
     <addaction name="actionmachine_human"/>
    </widget>
    <addaction name="actionSend"/>
    <addaction name="actionExit"/>
    <addaction name="menuchange"/>
   </widget>
   <widget class="QMenu" name="menuMointor">
    <property name="title">
     <string>Mointor</string>
    </property>
    <addaction name="actionVrep"/>
    <addaction name="actionserial"/>
    <addaction name="actionControl"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPyquino_help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMointor"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSend">
   <property name="text">
    <string>send Gcode</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionVrep">
   <property name="text">
    <string>Vrep</string>
   </property>
  </action>
  <action name="actionserial">
   <property name="text">
    <string>SerialPort</string>
   </property>
  </action>
  <action name="actionControl">
   <property name="text">
    <string>Control</string>
   </property>
  </action>
  <action name="actionPyquino_help">
   <property name="text">
    <string>Pyquino help</string>
   </property>
  </action>
  <action name="actionbasic">
   <property name="text">
    <string>basic</string>
   </property>
  </action>
  <action name="actionmachine_human">
   <property name="text">
    <string>machine human</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
